src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -494,4 +494,7 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
     <value>Repository Signatures resouce must be served over HTTPS. Source: {0}</value>
     <comment>{0} source service index</comment>
   </data>
+  <data name="Plugin_DownloadNotSupportedSinceUnsignedNotAllowed" xml:space="preserve">
+    <value>Downloading a package from a plugin is not supported since unsigned packages are not allowed and plugings do not support package signing.</value>
```

Comment:
"...and package download plugins do not support signed package verification."

Note the typo fix.